pkg/services/object/put/distributed.go

```diff
@@ -232,7 +232,7 @@ func (t *distributedTarget) distributeObject(obj objectSDK.Object, objMeta objec
 		return err
 	}
 
-	if t.localNodeInContainer && t.metainfoConsistencyAttr != "" {
+	if !t.localOnly && t.localNodeInContainer && t.metainfoConsistencyAttr != "" {
```

Comment:
looks correct now. We can now catch `t.localOnly` case before `defer` at the func's top, right?